### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by gbataille**
Some parts are confusing, unprecise, or mixing several concepts in one go which makes it hard to follow

### CODE REVIEW

**Comment by gbataille** [comment]
what first example are you talking about?

File: Readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 >>> age = 54
 
 >>> [fmt|Person's name is {name}, age is {age:x}|]
-"Person's name is Dave, age is 36"
+"Person's name is Dave, age is 54"
```
Comment (guibou) [inline Readme.md]: This is wrong. 54 in hexa is 36.

Comment (gbataille) [reply Readme.md]: ohh.... there is an hexa modificator in the very first example, without hexa prefix, before it's even mentioned...
ok, I'll fix the pattern then

**Comment by gbataille** [comment]
This one you have to merge yourself, I don't have commit rights :D

**Comment by guibou** [comment]
Thank you! I gave you commit right because of your recent contributions.
